services/app-web/src/components/FilterAccordion/FilterDateRange/FilterDateRange.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../../styles/uswdsImports.scss';
+@import '../../../styles/custom.scss';
+
+.dateRangePicker {
+  [class^='usa-label'] {
+    font-weight: normal;
+  }
+  [class^='usa-form-group'] {
+    margin-top: 0.75rem;
+    width: 100%;
+  }
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  column-gap: 32px;
+}
+
+[class='usa-fieldset'] {
+  legend {
+    font-weight: normal;
```

Comment:
Good use of targeted scss here.

Design nit: Please close the vertical spacing on the label - in the [designs](https://www.figma.com/file/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?type=design&node-id=138-9433&mode=design&t=ThvwlqwoeXAVG1zJ-0) the date range label is closer to the hint text which gives it more space between the inputs above
![Screenshot 2023-11-07 at 10 02 51 AM](https://github.com/Enterprise-CMCS/managed-care-review/assets/10750442/97525b35-6c19-47f8-a6a7-9a10046c0548)

Reply:
Design nit: Feels like we should have
![Screenshot 2023-11-07 at 10 09 08 AM](https://github.com/Enterprise-CMCS/managed-care-review/assets/10750442/c4e5abe4-8a24-4526-8d74-9d1c88cb813a)
 both inputs slide down when there is an error on either side. Keep them in alignment.